src/features/collapsible-sidebar/CollapsibleSidebarLogo.js

```diff
@@ -66,7 +71,7 @@ function CollapsibleSidebarLogo(props: Props) {
                 expandedElement={
                     <>
                         {toggleButton}
-                        <LinkBase data-resin-target={resinTarget} href="/">
+                        <LinkBase data-resin-target={resinTarget} href={linkUrl} {...linkHtmlAttributes}>
```

Comment:
Does this need to be a LinkBase or should it be possible to use a RouterLink here (to avoid full page refresh)?

Reply:
discussed offline and decided to use LinkBase as this can be swapped with RouterLink by passing in `RouterLink` as `component` prop